pyanaconda/localization.py

```diff
@@ -277,8 +280,22 @@ def setup_locale(locale, localization_proxy=None, text_mode=False):
         setenv("LANG", locale)
         locale_mod.setlocale(locale_mod.LC_ALL, locale)
 
+    set_modules_locale(locale)
+
     return locale
 
+
+def set_modules_locale(locale):
+    log.debug("setting modules locale to: %s", locale)
+    boss_proxy = BOSS.get_proxy()
+    modules = boss_proxy.GetModules()
+    for service_name in modules:
+        namespace = get_namespace_from_name(service_name)
+        object_path = get_dbus_path(*namespace)
+        module_proxy = DBus.get_proxy(service_name, object_path)
+        module_proxy.SetLocale(locale)
```

Comment:
We will probably need to set up the locale in the boss module as well, so this code might be moved to its method `set_locale`. Then you can call here just `boss_proxy.SetLocale(locale)`.

Reply:
Done